README.md

```diff
@@ -208,59 +208,85 @@ plaidPromise.then(successResponse => {
 });
 ```
 
-Plaid API methods that may return an MFA response pass an array of responses
-to the next promise. The first element of the array is the `mfaResponse`, and
-the second element is the `successResponse`. You can use destructuring to
-improve readability.
-
 For example:
 
 ```javascript
+'use strict';
+
+const bodyParser = require('body-parser');
+const express = require('express');
 const plaid = require('plaid');
 
-const plaidClient = new plaid.Client(CLIENT_ID, SECRET, PUBLIC_KEY, plaid.environments.sandbox);
+const plaidClient = new plaid.Client(
+  process.env.PLAID_CLIENT_ID,
+  process.env.PLAID_SECRET,
+  process.env.PUBLIC_KEY,
+  plaid.environments.sandbox
+);
+
+const app = express();
+const port = process.env.PORT || 3000;
+
+app.use(bodyParser.urlencoded({
+    extended: true
+}));
+app.use(bodyParser.json());
+
+app.post('/plaid_exchange', (req, res) => {
```

Comment:
i think this would be cleaner as a coroutine but maybe we shouldn't be so opinionated in our examples :)